native_client/tflitemodelstate.cc

```diff
@@ -111,9 +175,20 @@ TFLiteModelState::init(const char* model_path)
     return DS_ERR_FAIL_INTERPRETER;
   }
 
+  LOGE("Trying to detect delegates ...");
+  delegates_ = get_delegates();
```

Comment:
`ModifyGraphWithDelegate` with a `TfLiteDelegatePtr` is supposed to take ownership of the delegate, so do we need to keep their references in `delegates_` here? Can't this be a local variable?

Reply:
Maybe this has changed since r1.15, ill triple check